cmd/ndm_daemonset/probe/addhandler.go

```diff
@@ -247,6 +257,43 @@ func (pe *ProbeEvent) addBlockDevice(bd blockdevice.BlockDevice, bdAPIList *apis
 	return nil
 }
 
+// addBlockDeviceLabels adds labels to the blockdevice resource
+func (pe *ProbeEvent) addBlockDeviceLabels(bd *blockdevice.BlockDevice) {
+	// get the list of labels to be added
+	labelList := pe.Controller.LabelList
+	if labelList != "" {
+		labels := strings.Split(labelList, ",")
+		bd.Labels = make(map[string]string, 0)
+		for _, label := range labels {
+			// Get the value of the label from blockdevice device attributes
+			labelValue := getField(bd.DeviceAttributes, label)
+
+			// If the label value is empty check for filesystem information
```

Comment:
check for filesystem information. Didnt get this comment.

Reply:
nvm. got it. 

`label` will have the struct field key of each item, which we will use based on label list.